### PR TITLE
Explain in README how to override the database

### LINKED ISSUE #10: Add the ability to override the default database settings
Drupal Suite uses an SQLite database by default, which is most convenience for quick stand-up for local development, but unlikely to be used in production. Per https://github.com/phenaproxima/drupal-suite/pull/9#issuecomment-2070096704, provide the ability to override the default during `composer create-project` and use MySQL or Postgres instead. Document the feature.

### PR DESCRIPTION
**Pull request by phenaproxima**
Fixes #10 by adding a Composer script to do the workaround. When Drush 13.x is tagged, we can require that and instead rely on its environment variable-based option overrides, and remove both the dependency on `yaml-cli` and the `drupal:set-db` script.

### CODE REVIEW

**Comment by phenaproxima** [comment]
Just tried the instructions and they don't actually work, since post-create-project-cmd runs even if nothing has been installed. D'oh!

File: README.md [hunk 1]
```diff
@@ -5,13 +5,26 @@
...
 This one command will install Starshot and open it in a web browser for you to play with. You'll get all the modules and themes listed below, pre-configured.
 
-## Included modules and themes
+#### Which database?
+By default, Starshot sets up a SQLite database, so that you don't need to do any extra setup to get started.
+
+But if you want to install into another database, you can use the following commands instead of the one above, substituting the real database URL instead of `mysql://drupal:drupal@localhost/starshot`
+
+```
+composer create-project drupal/starshot-project --repository='{"type":"vcs","url":"https://github.com/phenaproxima/starshot-prototype"}' --stability=dev --no-install
```
Comment (TravisCarden) [inline README.md:20]: > the post-create-project-cmd script runs even if nothing has been installed.

@phenaproxima we can prevent that with `--no-scripts`. I tested the following, and it worked for me.

```suggestion
composer create-project drupal/starshot-project --repository='{"type":"vcs","url":"https://github.com/phenaproxima/starshot-prototype"}' --stability=dev --no-install --no-scripts
```

**Comment by phenaproxima** [comment]
The situation in `main` has dramatically changed relative to this PR.

So first of all, we're now installing via `core/scripts/drupal install`, which hard-codes the use of SQLite with no way to override. That will need to be fixed in core, full stop.

**Comment by phenaproxima** [comment]
I've thought of a workaround here -- let's create a command that can use `drush sql:dump` and whatnot export the SQLite database, and import it into another database.

**Comment by phenaproxima** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr27-1ifryowatgfciftcfr4rm5bla0yqdxtj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/663b97fd1c897942d7d7f1b5

**Comment by phenaproxima** [comment]
[Tugboat](https://www.tugboatqa.com) has finished building the preview for this pull request!

Link:
- https://pr27-1ifryowatgfciftcfr4rm5bla0yqdxtj.tugboatqa.com


Dashboard:
- https://dashboard.tugboatqa.com/663b97fd1c897942d7d7f1b5

**Pull request closed without merging**